day10/ex1.py

```
@@ -0,0 +1,21 @@
+    Returns True if the given year is a leap year, otherwise False.
+    A leap year is divisible by 4, but not by 100 unless also divisible by 400.
+    """
+    if year % 4 != 0:
+        return False
+    if year % 100 != 0:
+        return True
+    if year % 400 == 0:
+        return True
+    return False
+#Solution 2
+def is_leap_year(year):
```

Ruff (0.12.2)

Redefinition of unused `is_leap_year` from line 2

(F811)